array-methods/for-each.ts

```diff
@@ -0,0 +1,21 @@
+const values = [10, 1, 22, 23, 41, 5, 18, 7, 80, 9];
+
+values.forEach((values) => {
+  console.log(values);
+});
+
+console.log('serparation');
+
+values.forEach((_, index) => {
+  const reverse = values.length - 1 - index;
+  console.log(reverse);
```

Comment:
This is just logging the indexes of the numbers. You should be logging the values that exist AT this index